big_tests/tests/mongoose_helper.erl

```diff
@@ -478,6 +479,10 @@ restore_sasl_mechanisms_config(Config) ->
 set_sasl_mechanisms(GlobalOrHostSASLMechanisms, Mechanisms) ->
     rpc(mim(), ejabberd_config, add_local_option, [GlobalOrHostSASLMechanisms, Mechanisms]).
 
+set_store_password(HostType, Type) ->
```

Comment:
Could we have just one function? It could use `domain_helper:host_type()`. For static domains it would just return `localhost`.